test/e2e/cluster_extension_install_test.go

```diff
@@ -332,16 +326,18 @@ func TestClusterExtensionInstallRegistryMultipleBundles(t *testing.T) {
 	t.Log("By eventually reporting a failed resolution with multiple bundles")
 	require.EventuallyWithT(t, func(ct *assert.CollectT) {
 		assert.NoError(ct, c.Get(context.Background(), types.NamespacedName{Name: clusterExtension.Name}, clusterExtension))
-		assert.Nil(ct, clusterExtension.Status.Resolution)
+		cond := apimeta.FindStatusCondition(clusterExtension.Status.Conditions, ocv1alpha1.TypeProgressing)
+		require.NotNil(ct, cond)
+		assert.Equal(ct, metav1.ConditionFalse, cond.Status)
+		assert.Equal(ct, ocv1alpha1.ReasonFailed, cond.Reason)
+		assert.Contains(ct, cond.Message, "in multiple catalogs with the same priority [operatorhubio test-catalog]")
```

Comment:
We already have a progressing condition check immediately below this. These checks also look incorrect based on the values that would be set by the progressing condition.

Reply:
fixed in https://github.com/operator-framework/operator-controller/commit/febe76e179455d20c593b3f5956a770835afcedc